### DISCUSSION TITLE
.map does not work as intended

### DISCUSSION THREAD

**Discussion by JohannesMisch** [neuefische/questions#460] [Web] [answered]
I want to map over an array (the array is stored in a db.json) to create a card for each object in this array. The first card is created as intended but some of the data for the second card is undefined, although I used the same key name in object one and object two.
example object:
```json
[
  {
    "id": 0,
    "device": "TV",
    "device_category": "Entertainment",
    "model": "LG 55NANO796PC LED TV",
    "power_consumption": 95,
    "power_consumption_standby": 0.5,
    "average_usage_time": 4,
    "location": "Livingroom"
  }
]
```
"power_consumption", "power_consumption_standby" and "average_useage_time" does not get displayed for the second card and the third card.
Link to the branch I am working on: [branch](https://github.com/JohannesMisch/Capstone-project/tree/createCard)

**Comment by mbosselmann** [comment, marked as answer]
We solved the problem together and the solution was to remove the data in the `localStorage` as this was not the same anymore as the example data from the `db.json`.

**Reply by JohannesMisch** [reply]
Thank you, Mareike and Felix, for your help.
